server/asset/common.go

```diff
@@ -75,21 +90,23 @@ type FundingCoin interface {
 	// SpendSize returns the size of the serialized input that spends this
 	// FundingCoin.
 	SpendSize() uint32
-	// Value is the output value.
-	Value() uint64
+}
+
+// Output is a transaction output.
+type Output interface {
+	Coin
+	Addresses() []string
 }
```

Comment:
We discussed eliminating this interface, just using Coin for coins that represent tx outputs.